test/api_integration/apis/saved_objects/find.js

```diff
@@ -46,6 +46,7 @@ export default function ({ getService }) {
                   attributes: {
                     title: 'Count of requests',
                   },
+                  score: 0,
```

Comment:
So, this is a little scary on the relevance of this PR to be honest. I had to add the `score` property on the `find` API FTRs, and it seems that every call we are performing in the suite returns a score of `0`... 

When testing from the saved object management table page, I did have some non-zero `score` value however, so I guess we might want/need to do some scoring tuning at some point. I think this is complex enough to be done as a follow-up though.

@rudolf any thoughts on that?

Reply:
Looks like we don't have any integration tests that actually specify a search query in the `search` query param. Would be great if we could add that (could be a separate PR).